config/bolt/contenttypes.yaml

```diff
@@ -418,6 +419,7 @@ tests:
 #
 #contenttype_abc:
 #    name: ContentType Abc
+#    slug: contenttypeabc
```

Comment:
I believe we're enforcing now that the `key` should be the same as the `slug`. So, i think the example should say: 

```yaml
#contenttypeabc:
#    name: ContentType Abc
#    slug: contenttypeabc
```

So, both _without_ the `_`.